tests/unit/xoopsLib/Xoops/Core/Kernel/Handlers/XoopsRanksHandlerTest.php

```diff
@@ -32,6 +32,13 @@ public function test___construct()
 		$this->assertSame('rank_title',$instance->identifierName);
     }
     
+    public function testContracts()
+    {
+        $instance=new $this->myclass($this->conn);
```

Comment:
Can you change this to $instance = $this->object;

$this->conn is not a defined property, and the type hint for the arg is Connection. This seems to work OK through PHP 5.6, but [triggers an error in PHP7.0](https://travis-ci.org/XOOPS/XoopsCore/jobs/79988983).

PHPUnit 4.8.6 creates another error trying to handle this one, and apparently that will be fixed in 5.0. Fixing the cause should make everything happy.